psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -826,6 +827,39 @@ void OEProp::compute() {
     if (tasks_.count("NO_OCCUPATIONS")) compute_no_occupations();
     if (tasks_.count("GRID_FIELD")) compute_field_over_grid();
     if (tasks_.count("GRID_ESP")) compute_esp_over_grid();
+    if (tasks_.count("PE_CORR")) compute_pe();
+}
+
+PePropCalc::PeOutputType PePropCalc::compute_pe_prop(bool transition, bool print_output, bool verbose) {
```

Comment:
I don't think this belongs here as a one-electron property. It also seems the only thing you need to compute the perturbative correction is the AO density matrix, hence this can possibly become a free function in the Python layer of the code?